site/de/index.html

```diff
@@ -0,0 +1,664 @@
+                        <ul class="dropdown-menu">
+                            <li><a href="javadoc/9.2.0">9.2.0</a></li>
+                            <li><a href="javadoc/9.1.0">9.1.0</a></li>
+                            <li><a href="javadoc/9.0.0">9.0.0</a></li>
+                            <li><a href="javadoc/8.2.0">8.2.0</a></li>
+                            <li><a href="javadoc/8.1.0">8.1.0</a></li>
+                            <li><a href="javadoc/8.0.0">8.0.0</a></li>
+                            <li><a href="javadoc/7.1.0">7.1.0</a></li>
+                        </ul>
+                    </li>
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                           aria-expanded="false">Sprachen<span class="caret"></span></a>
+                        <ul class="dropdown-menu">
+                            <li><a href="../index.html">English</a></li>
+                            <li><a href="">Deutsch</a></li>
+                            <li><a href="../cn/index.html">中文</a></li>
+                            <li><a href="../fr/index.html">Français</a></li>
+                            <li><a href="../ja/index.html">日本語</a></li>
+                            <li><a href="../pt-br/index.html">Português-Brasil</a></li>
+                            <li><a href="../ru/index.html">Русский</a></li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+
+
+    <!-- Header -->
+    <a class="anchor" name="about"></a>
+    <div class="intro-header">
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-2 col-lg-8">
+                    <div class="intro-message">
+                        <h1>Eclipse <span style="font-weight: 400">Collections</span></h1>
+                        <h3>- Features die du möchtest, mit Collections die du brauchst -</h3>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- /.intro-header -->
+
+    <!-- Page Content -->
+
+    <a class="anchor" name="start"></a>
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Einführung in die <br>Eclipse Collections</h2>
+                    <p class="lead">Eclipse Collections ist das beste Java collections Framework um Freude
+                        in die Java Entwicklung zu bringen.
+                        <br><br>Integriere sie heute noch in deinen Source Code mit deinem favorisierten Build-Tool!</p>
```

Comment:
Change 'sie' to 'es'
Integriere es
(es geht um _das_ Framework, das zu integrieren ist)